<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>waiter</class>
 <widget class="QDialog" name="waiter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务员</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-image: url(:/image1/图片/640a102f1dee72f8.jpg);}
font: 75 12pt &quot;楷体&quot;;
QPushButton{
	background-color: rgb(85, 255, 0);}
font: 75 12pt &quot;楷体&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;
background-color: rgb(25, 136, 255);</string>
       </property>
       <property name="text">
        <string>已点菜单</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;
background-color: rgb(25, 136, 255);</string>
       </property>
       <property name="text">
        <string>做菜进度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;
background-color: rgb(25, 136, 255);</string>
       </property>
       <property name="text">
        <string>刷新信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QStackedWidget{
	background-image: url(:/image1/图片/636f0b10610705a3.jpg);

}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableView" name="tableView_2">
         <property name="styleSheet">
          <string notr="true">QTableView{
	
background-image: url(:/image1/图片/d84df8aacf4e21bd.jpg);}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="styleSheet">
          <string notr="true">QTableView{
	background-image: url(:/image1/图片/d84df8aacf4e21bd.jpg);}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 14pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>顾客实时需求</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 255, 255);
font: 75 12pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>收到信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView_3">
       <property name="styleSheet">
        <string notr="true">QTableView{background-image: url(:/image1/图片/95131d2be7c43cb8.jpg);
	
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>消费：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>收到：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>元</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>找零：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>元</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);
font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>现金结算</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);
font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>微信支付</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);
font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>支付宝</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 255);
font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
